From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Axel Garcia K." <git@axelgk.com>
Date: Tue, 5 May 2026 18:57:45 +0000
Subject: [PATCH 2/5] Include time header for qpsx XA code

Include sys/time.h where the XA audio code uses timeval so HCRTOS newlib sees the matching type declarations.
---
 src/spu/spu_pcsxrearmed/xa.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/spu/spu_pcsxrearmed/xa.c b/src/spu/spu_pcsxrearmed/xa.c
index 2edcfb0704bf6e05165d277e78307502ffffa6c6..6c4637d7ea9faace0922b8664faa50e2b1d21ce0 100644
--- a/src/spu/spu_pcsxrearmed/xa.c
+++ b/src/spu/spu_pcsxrearmed/xa.c
@@ -18,6 +18,7 @@
 #include "stdafx.h"
 #define _IN_XA
 #include <stdint.h>
+#include <sys/time.h>
 
 // will be included from spu.c
 #ifdef _IN_SPU
-- 
2.54.0

